test/integration/test_seq_copy_int.py

```diff
@@ -42,9 +42,9 @@
 _TEST_MAX_LENGTH = 20
 
 # tuple format: (train_params, translate_params, use_prepared_data, use_source_factors)
-ENCODER_DECODER_SETTINGS = [
+ENCODER_DECODER_SETTINGS_TEMPLATE = [
     # Basic transformer, nbest=2 decoding
-    ("--encoder transformer --decoder transformer"
+    ("--encoder transformer --decoder {decoder}"
```

Comment:
what is the additional runtime of integration tests with this cross-product of test x decoder types?

Reply:
Unit/integration tests last 15-20s with both, so I don't think that it's significant. The system tests, however, take about 8 minutes for each decoder.